tardis/plugins/auditor.py

```diff
@@ -85,7 +89,21 @@ async def notify(self, state: State, resource_attributes: AttributeDict) -> None
                 .replace(tzinfo=self._local_timezone)
                 .astimezone(pytz.utc)
             )
-            await self._client.update(record)
+            try:
+                await self._client.update(record)
+            except RuntimeError as e:
+                reqwest_error_match = REQWEST_ERROR.match(str(e))
+                if reqwest_error_match:
+                    http_error_code = reqwest_error_match.group(1)
+                    if http_error_code == "400":
```

Comment:
Yes, I think you are right. This might need a little bit of time to get fixed in Auditor due to the holiday season. I've opened a new issue https://github.com/ALU-Schumacher/AUDITOR/issues/378. I plan to include this change in the v0.2.0 release of Auditor. Once released, we will create a PR for tardis that includes the version update and updates to all breaking changes.